standard/classes.md

```diff
@@ -4460,7 +4460,7 @@ binary_operator_declarator
 
 overloadable_binary_operator
     : '+'  | '-'  | '*'  | '/'  | '%'  | '&' | '|' | '^'  | '<<' 
-    | right_shift | '==' | '!=' | '>' | '<' | '>=' | '<='
```

Comment:
This change is wrong. The *right_shift* production exists to resolve a syntactic ambiguity in C#.